profiles/test_speed_unit.lua

```diff
@@ -0,0 +1,22 @@
+print('hello world')
```

Comment:
Is it a local test file?  If yes please remove, if not, please:
- please remove upper line
- please add test cases related your new logic